components/ui/partner-card.js

```diff
@@ -1,41 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-class PartnerCard extends React.Component {
-  static propTypes = {
```

Comment:
Why did you remove these static props and move them to the bottom? I know that every other component has them at the bottom. But I think that is more readable and more usable to have them here. Whenever you enter a component you will know which props are allowed and the type of them

Reply:
Happy to move `propTypes` back to a static property. I agree, the component is more understandable with the `propTypes` up front. I was aligning to the other components I had encountered - consistency across components important to readability and usability too.

I'll move back and ensure new components have static property for `propTypes`.

Reply:
Yes, let's do that whenever we can